app/controllers/watchdog/logging_api_controller.rb

```diff
@@ -18,10 +18,13 @@ def active_users_monthly
 
     private
 
-    def extract_data_for period
-      f = File.read(File.join(Rails.root, 'tmp/data/active_users.json'))
+    def extract_data_for(period)
+      raw = S3_BUCKET.get_object(
+        bucket: 'govwifi-mock-metrics',
+        key: 'active_users.json'
+      )
 
-      data = JSON.parse(f)
+      data = JSON.parse(raw.body.read)
```

Comment:
How is this going to behave if S3 returns an error?

Reply:
The widget will display `Error Loading Chart: Internal Server Error`

Reply:
I guess that's good enough for now.